samples/bluetooth/mesh/light/src/smp_dfu.c

```diff
@@ -142,6 +142,8 @@ int smp_service_adv_init(void)
 
 int smp_dfu_init(void)
 {
+	int err;
```

Comment:
Do you need this at all? As far as I can see this can be just `return smp_service_adv_init();` now.